spatialpandas/dask.py

```diff
@@ -581,10 +585,8 @@ def __getitem__(self, key):
         return result
 
 
-@make_meta_obj.register(GeoDataFrame)
+@make_meta_dispatch.register(GeoDataFrame)
```

Comment:
This was wrongly changed to obj in https://github.com/holoviz/spatialpandas/pull/171